src/index.js

```diff
@@ -56,6 +56,10 @@ export default function (options = {}, callback = function noop () {}) {
 
     if (specFiles.length) {
       protractorArgs = protractorArgs.filter((arg) => !/^--suite=/.test(arg))
+      var specIndex = protractorArgs.indexOf('--specs')
+      if (specIndex > -1) {
```

Comment:
It's a little dirty (since the second part of the arg will stay around in the args passed, but could we just re-use the test above and include both `--specs` and `--suite`?

e.g.

```
protractorArgs.filter((arg) => !/^--(suite|specs)/.test(arg))

```